### PR TITLE
Box2D Testbed improvement

### PR DESCRIPTION
**Pull request by aismann**

### CODE REVIEW

**Comment by aismann** [comment]
@halx99, what is wrong?
Any hints for me?

File: cocos/2d/CCDrawNode.h [hunk 1]
```diff
@@ -215,7 +215,21 @@ class CC_DLL DrawNode : public Node
...
+      * @param angle  The circle angle.
+      * @param segments The number of segments.
+      * @param scaleX The scale value in x.
+      * @param scaleY The scale value in y.
+      * @param fillColor The color will fill in polygon.
+      * @param borderWidth The border of line width.
+      * @param borderColor The border of line color.
+      * @js NA
+      */
+    void DrawNode::drawSolidCircle(const Vec2& center, float radius, float angle, unsigned int segments, float scaleX, float scaleY, const Color4F& fillColor, float borderWidth, const Color4F& borderColor);
```
Comment (halx99) [inline cocos/2d/CCDrawNode.h:231]: Please remove `DrawNode::` to fix the non-windows ci

Comment (aismann) [reply cocos/2d/CCDrawNode.h]: Thanks. I have overseen it.

**Comment by aismann** [comment]
Short information:
I think the most stuff is done.
Not all TestBed demos working full now (because of different keyboard implementation).
I will not longer spend more time for the testbed stuff.
Need to work on my own projects again. 
But If I find a 'bug' on the 'DebugDraw' or 'PhysicsDebugNodeChipmunk2D' classes I will fix it.

**Comment by aismann** [comment]
@halx99 
It this a OSX Problem or a problem of the testbed sources?

**Comment by halx99** [comment]
The build still failed on linux and macOS

**Comment by aismann** [comment]
I  will look later.

**Pull request closed without merging**